spring-boot-admin-server-ui/src/main/frontend/views/applications/index.vue

```diff
@@ -31,6 +31,13 @@
       </div>
       <template v-if="applicationsInitialized">
         <applications-stats :applications="applications" />
+            <input
+              class="input"
+              type="search"
+              placeholder="Application filter"
+              v-model="filter"
+              v-if="applications.length>1"
+            >
```

Comment:
maybe add button to clear the filter in a common container
```
<div v-if="applications.length>1">
  <input  class="input" type="search" placeholder="Application filter" v-model="filter">
  <button v-show="this.filter!==''" @click="this. filter = ''">Clear</button>
</div>
```

Reply:
at right side of textbox there is big X. so this is covered.
![image](https://user-images.githubusercontent.com/604405/51800774-e2c95580-2234-11e9-8677-1ea0a0adcbaf.png)